047-TrafficControlWithDapr/Student/Challenge-00.md

```diff
@@ -0,0 +1,323 @@
+    
+*Your IT organization may provide you access to an Azure resource group, but not the entire subscription. If that's the case, take note of that resource group name and make sure you have `Contributor` access to it, using the instructions mentioned above.*
+
+Next, you'll create the Azure resources for the subsequent challenges using [Azure Bicep](https://docs.microsoft.com/azure/azure-resource-manager/bicep/overview) and the [Azure CLI](https://docs.microsoft.com/cli/azure/what-is-azure-cli).
+
+1.  If you're using [Azure Cloud Shell](https://shell.azure.com), skip this step and proceed to step 2. Open the [terminal window](https://code.visualstudio.com/docs/editor/integrated-terminal) in VS Code and make sure you're logged in to Azure
+
+    ```shell
+    az login
+    ```
+
+1.  Make sure you have selected the Azure subscription in which you want to work. Replace the 'x's with your subscription GUID or subscription name. The subscription GUID can be found in the Azure Resource Group blade from the Azure Portal.
+
+    ```shell
+    az account set --subscription "xxxx-xxxx-xxxx-xxxx"
+    ```
+
+1.  Generate an [SSH key pair](https://docs.microsoft.com/azure/virtual-machines/linux/ssh-from-windows#overview-of-ssh-and-keys) if you don't already have one.
+
+    ```shell
+    ssh-keygen -t rsa -b 2048
+    ```
+    - If prompted for a file name, leave the entry blank, and press enter.
+    - If prompted for a passphrase, leave the entry blank, and press enter.
+   
+    Once complete, you'll find two SSH key files in the following directory: `%USERPROFILE%\.ssh`. Right-click on the `id_rsa` file and open with Notepad. Copy the entire contents of the file which is the public key. You'll need it to configure the parameter file in an upcoming step.
+
+1.  In the accompanying source code, modify the `Resources/Infrastructure/bicep/main.parameters.json` file so it contains the proper data for the deployment:
+
+    ```json
+    {
+      "appName": {
+         "value": "dapr"
+      },
+      "region": {
+         "value": "southcentralus"
+      },
+      "environment": {
+         "value": "youruniqueid123"
+      },
+      "adminUserName": {
+         "value": "adminbruce"
+      },
+      "publicSSHKey": {
+         "value": "ssh-rsa AAAAB...wnBTn bruce.wayne@<your computer name>"        
+      }
+    }
+    ```
+
+1.  Create a new resource group for your WhatTheHack using the `Resources/Infrastructure/bicep/rg.bicep` script file. 
+
+    **When invoking the command, replace the `location` argument with the Azure region you want to use.**
+
+    ```shell
+    cd ./Resources/Infrastructure/bicep/
+    az deployment sub create --location "<resource-group-location>" --template-file rg.bicep --parameters ./main.parameters.json --query "properties.outputs" --output yamlc
```

Comment:
Do you have a typo for --output yaml? Says --output yamlc

Reply:
Found out after I put this in that there is a yamlc. :) The other one in this doc is not yamlc? Perhaps both should either be yamlc or yaml to be consistent? Up to you